src/extension.ts

```diff
@@ -3,15 +3,26 @@
 import * as vscode from 'vscode';
 import * as AWS from 'aws-sdk';
 import { DataFrame } from 'dataframe-js';
-import { listColumns, listDatabases, listTables, runAthenaQuery } from './athena';
 import { Query } from './query';
 import { parse_queries } from './utils';
-import { countReset } from 'console';
+import {
+	getRunQueryFunction,
+	getListColumnsFunction,
+	getListTablesFunction,
+	getListDatabasesFunction
+} from './db-selection';
+import { selectActiveConn, addConn, deleteConn } from './connection'
+
 
 AWS.config.update({
 	region: 'us-east-1'
 });
 
+const runQuery = getRunQueryFunction();
+const listDatabases = getListDatabasesFunction();
+const listTables = getListTablesFunction();
+const listColumns = getListColumnsFunction();
+
```

Comment:
TODO: need to move this selection into the individual functions so that it's reloaded if the active DB changes